force-app/main/default/lwc/givingSummary/givingSummary.html

```diff
@@ -19,12 +19,14 @@ <h2 class="slds-text-heading_small slds-var-p-left_large">
       </template>
       <template if:false={errorMessage}>
         <lightning-layout-item
-          size="12"
-          small-device-size="12"
+          size="4"
```

Comment:
Couldn't we just use `size="4" `since they're all the same? Worth testing.

Reply:
Done!